docs/user/visualize.asciidoc

```diff
@@ -3,86 +3,85 @@
 
-<<TSVB,TSVB>>:: Visualizes time series data using pipeline aggregations.
+a| <<TSVB,*TSVB*>>
 
-<<timelion,Timelion>>:: Computes and combine data from multiple time series
-data sets.
+Display and compare your time series data by combining an infinite number of data aggregations with TSVB visualizations, including metric, Top N, gauge, markdown, and table. 
+With TSVB, you can analylze multiple index patterns and customize every aspect of your visualization. 
 
-<<visualize-maps, Maps>>::
-* *<<maps,Elastic Maps>>* &mdash; Displays geospatial data in {kib}.
+| <<timelion,*Timelion*>>
 
-* <<heat-map,Heat map>>:: Display shaded cells within a matrix.
+Create two-dimensional graphs with your time series data using Timelion, which is driven by a simple expression language you use to retrieve data, perform calculations, and visualize the results.
 
-<<for-dashboard,Dashboard tools>>::
+a| <<maps,*Elastic Maps*>>
 
-* *Markdown widget* &mdash; Displays free-form information or instructions.
+Parse through your location-based data to create maps with multiple layers and data sources, map individual geo points and shapes, and use dynamic client-side styling.
 
-* *Controls* &mdash; Adds interactive inputs to a dashboard.
+| <<heat-map,*Heap map*>>
 
-<<vega-graph,Vega>>:: Completes control over query and display.
+Create visualizations of your categorical data, where the inidividual values are represented by colors. 
 
-[float]
-[[choose-your-data]]
-== Choose your data
+| <<for-dashboard, *Dashboard tools*>>
 
-Specify a search query to retrieve the data for your visualization, or used rolled up data.
+Build tools that enable you to interact with the data on your dashboards in real time.
 
-* To enter new search criteria, select the <<index-patterns, index pattern>> for the indices that
-contain the data you want to visualize. The visualization builder opens
-with a wildcard query that matches all of the documents in the selected
-indices.
+a| <<vega-graph,*Vega*>>
 
-* To build a visualization from a saved search, click the name of the saved
-search you want to use. The visualization builder opens and loads the
-selected query.
-+
-NOTE: When you build a visualization from a saved search, any subsequent
-modifications to the saved search are reflected in the
-visualization. To disable automatic updates, delete the visualization
-on the *Saved Object* page.
+Create fully customizable visualizations with multiple data sources that are available from the Vega library.
 
-* To build a visualization using <<rollup-index-pattern,rolled up data>>, select
-the index pattern that includes the data. Rolled up data is summarized into
-time buckets that can be split into sub buckets for numeric field values or
-terms. To lower granularity, use a time aggregation that uses and combines
-several time buckets. For an example, refer to <<rollup-data-tutorial,Create and visualize rolled up data>>.
+|===
 
 [float]
 [[vis-inspector]]
 == Inspect visualizations
```

Comment:
There are a lot of different formats on this page.  It might be nice to put a page break between the overview info and the tasks.  Or maybe put this at the end of the visualize section.